pkg/services/provisioning/provisioning.go

```diff
@@ -127,6 +129,10 @@ func (ps *provisioningServiceImpl) ProvisionDashboards() error {
 	return nil
 }
 
+func (ps *provisioningServiceImpl) GetDashboardFileReaderByName(name string) *dashboards.FileReader {
```

Comment:
Main change here. I needed to get the path from the provisioner config based on the name of the provisioner in the database. This seemed to be like the best place to have that as this service already has reference to dashboardProvisioner and it to each filerReader with config.